### ISSUE TITLE
Any way we can make it look like svg/canvas based word-cloud

### ISSUE THREAD

**Issue by sahil-isima** [madox2/react-tagcloud#25]
**Note**: Not a functional issue. More of a style enhancement.

I like this package as its light weight and provide flexibility. For most data input, it gives decent output for word cloud but for certain type of input, output doesn't seems to look like word cloud. 

Example : https://codesandbox.io/s/white-forest-h4zd4

**Comment by madox2** [comment]
Hi, thanks for your feedback! I am not sure what you are trying to accomplish, could describe/attach an example how you would expect the cloud to look like?

In the sandbox code you shared is cloud displayed on a single line, maybe setting a fixed width would make it look better. (e.g. `width: 200`)

**Comment by sahil-isima** [comment]
Thanks for suggestion. I tried and now it looks like :
https://codesandbox.io/s/white-forest-h4zd4?file=/src/App.js

Than i tried same dataset with react-wordcloud and it looks like : 
https://codesandbox.io/s/react-wordcloud-responsive-forked-pyjbi?file=/src/index.js

I understand the fact that they are using svg/canvas based positioning while you are using normal html based technique. You seems to have kept it simple which gives you a performance advantage.

**Comment by madox2** [comment]
That's right, this tag cloud component is supposed to be rather simple - html + css is used to position tags. You can provide your own css classes but I am afraid it would be complicated to make it look like your first picture. My suggestion is if react-wordcloud library works better for you then use it. Also if you find a way how to improve on this PRs are very welcome
